js/gratipay/team.js

```diff
@@ -192,6 +191,7 @@ Gratipay.team = (function() {
                     if(d.success) {
                         Gratipay.notification(d.success, 'success');
                     }
+                    debugger;
```

Comment:
Could be. I know that Chrome ignores `debugger` statements when the developer tools are not open.